targets/lenovo-x1-carbon.nix

```diff
@@ -348,6 +357,10 @@
                   [netvmPCIPassthroughModule]
                   ++ netvmExtraModules;
               };
+              virtualization.microvm.idsvm = {
+                enable = true;
```

Comment:
Let's set this to false by default

Reply:
Set to false by default in [b500944](https://github.com/tiiuae/ghaf/commit/b500944e5baef64e8efefa4b2e5fbbedcdbbbce5)
as well as mitmproxy.